Fix docker badge

![image](https://user-images.githubusercontent.com/3787410/125157414-6a08aa80-e19d-11eb-88e0-e8b92d76737e.png)

@hsluoyz Build failed
```
closetool@Ubuntu-20:~/tasks/casbin-server$ docker build .
Sending build context to Docker daemon  471.6kB
Step 1/13 : FROM golang:1.12
 ---> ffcaee6f7d8b
Step 2/13 : RUN apt-get update &&     apt-get -y install git unzip build-essential autoconf libtool
 ---> Using cache
 ---> 99b7a5384050
Step 3/13 : RUN git clone https://github.com/google/protobuf.git &&     cd protobuf &&     ./autogen.sh &&     ./configure &&     make &&     make install &&     ldconfig &&     make clean &&     cd .. &&     rm -r protobuf
 ---> Using cache
 ---> 2d1656e84736
Step 4/13 : RUN go get google.golang.org/grpc
 ---> Running in e01e74df6df4
# golang.org/x/net/http2
src/golang.org/x/net/http2/client_conn_pool.go:305:6: undefined: errors.Is
The command '/bin/sh -c go get google.golang.org/grpc' returned a non-zero code: 2
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.12
+FROM golang:1.13
 
 RUN apt-get update && \
     apt-get -y install git unzip build-essential autoconf libtool

--- a/README.md
+++ b/README.md
@@ -2,7 +2,7 @@ Casbin Server
 ====
 
 [![Build Status](https://travis-ci.org/casbin/casbin-server.svg?branch=master)](https://travis-ci.org/casbin/casbin-server)
-[![Docker](https://img.shields.io/docker/build/casbin/casbin-server.svg)](https://hub.docker.com/r/casbin/casbin-server/builds/)
+[![Docker](https://img.shields.io/docker/cloud/build/casbin/casbin-server)](https://hub.docker.com/r/casbin/casbin-server/builds/)
 [![Coverage Status](https://coveralls.io/repos/github/casbin/casbin-server/badge.svg?branch=master)](https://coveralls.io/github/casbin/casbin-server?branch=master)
 [![Godoc](https://godoc.org/github.com/casbin/casbin-server?status.svg)](https://godoc.org/github.com/casbin/casbin-server)
 

--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module github.com/casbin/casbin-server
 
-go 1.12
+go 1.16
 
 require (
 	github.com/casbin/casbin/v2 v2.0.2

--- a/proto/casbin.proto
+++ b/proto/casbin.proto
@@ -17,6 +17,7 @@ syntax = "proto3";
 option java_multiple_files = true;
 option java_package = "io.grpc.examples.proto";
 option java_outer_classname = "CasbinProto";
+option go_package="./;proto";
 
 package proto;
 

